### PR TITLE
Add copy to Workspace Limits page

### PR DESCRIPTION
**Pull request by emdecr**
#### Summary
This PR adds legal copy to the Workspace Limits page.

Directive was to add the following sentence before the FAQ: 

> Mattermost reserves the right to archive or delete messages and establish or change (1) limits as to how many messages can be stored, (2) how long such messages will be stored, and (3) charges for storing such messages. Mattermost further reserves the right to establish or change limits on periods of inactivity that may result in the termination of your messages cloud storage and deletion or archiving of any stored messages.

@jasonblais Changes as per the thread with Nirosha. Let me know if we want to differentiate the new copy in any way (ex. putting it in a Note section as seen further up on the page where it says `These limits do not apply to self-hosted deployments.`, or changing the text styling, etc.

cc: @cwarnermm for general visibility, and in case there's already guidelines for how to treat this kind of copy.

Thanks!

### CODE REVIEW

File: source/onboard/mattermost-limits.rst [hunk 1]
```diff
@@ -36,6 +36,8 @@ Mattermost Professional limits
 
 - 250GB file storage limit.
 
+Mattermost reserves the right to archive or delete messages and establish or change (1) limits as to how many messages can be stored, (2) how long such messages will be stored, and (3) charges for storing such messages. Mattermost further reserves the right to establish or change limits on periods of inactivity that may result in the termination of your messages cloud storage and deletion or archiving of any stored messages.
```
Comment (jasonblais) [inline source/onboard/mattermost-limits.rst:39]:
```suggestion
.. note::

   Mattermost reserves the right to archive or delete messages and establish or change (1) limits as to how many messages can be stored, (2) how long such messages will be stored, and (3) charges for storing such messages. Mattermost further reserves the right to establish or change limits on periods of inactivity that may result in the termination of your messages cloud storage and deletion or archiving of any stored messages.
```

**Review by jasonblais — CHANGES_REQUESTED** [review]
@emdecr slight proposed adjustment to format it as a note.

Also, are you open to moving this above the "Mattermost Professional limits" section, since it's specifically about the message history limit.

Thanks for the quick update!

**Comment by emdecr** [comment]
@jasonblais Sure thing! Just pushed up an edit for the placement and note styling.

Screen Shot 2022-08-10 at 8 55 23 AM

**Review by jasonblais — APPROVED** [review]
Thanks!

**Pull request merged**